### ISSUE TITLE
DBbackup can't work in python3 because we use File Object attribute set

### ISSUE THREAD

**Issue by ZuluPro** [django-dbbackup/django-dbbackup#89]
In Python3, I skip a lot of test using dbbackup because we use attribute setting:

```
======================================================================
ERROR: test_encrypt (dbbackup.tests.functionnal.test_commands.DbBackupCommandTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/mock.py", line 1201, in patched
    return func(*args, **keywargs)
  File "/home/zulu/projects/django-dbbackup/dbbackup/tests/functionnal/test_commands.py", line 26, in test_encrypt
    execute_from_command_line(argv)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/__init__.py", line 399, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/__init__.py", line 392, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/base.py", line 242, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/base.py", line 285, in execute
    output = self.handle(*args, **options)
  File "/home/zulu/projects/django-dbbackup/dbbackup/utils.py", line 80, in wrapper
    func(*args, **kwargs)
  File "/home/zulu/projects/django-dbbackup/dbbackup/management/commands/dbbackup.py", line 55, in handle
    self.save_new_backup(database)
  File "/home/zulu/projects/django-dbbackup/dbbackup/management/commands/dbbackup.py", line 68, in save_new_backup
    outputfile.name = filename
AttributeError: can't set attribute
```

**Comment by ZuluPro** [comment]
We should remove all ``fileobj.name = foo``, it's not compatible with Python 3.

**Comment by ZuluPro** [comment]
I made a fix with PR #96.

**Comment by benjaoming** [comment]
I present you the :trophy: of most issues closed in 2015 :)
